src/Sarif/CodeGenHints.json

```diff
@@ -114,8 +114,8 @@
       "arguments": {
         "typeName": "AlgorithmKind",
         "description": "Values specifying different hashing algorithms.",
-        "zeroValue": "Unknown",
-        "enumValues": [
+        "zeroValueName": "Unknown",
+        "enumNames": [
```

Comment:
Yes, would be better. Requires a JSchema code generator change so will handle in a separate PR.

---
In reply to: [62932360](https://github.com/Microsoft/sarif-sdk/pull/259#discussion_r62932360) [](ancestors = 62932360,62882321)